build.gradle

```diff
@@ -63,7 +47,6 @@ dependencies {
 
     compile('org.codehaus.groovy.modules.http-builder:http-builder:0.7.1') {
         exclude module: "commons-logging"
-        exclude module: "xml-apis"
```

Comment:
To get the tests to pass I had to remove this line, though not sure if you still want it excluded.